electron-app/vue.config.js

```diff
@@ -1,5 +1,6 @@
 // vue.config.js
 module.exports = {
+  productionSourceMap: false,
```

Comment:
the warnings are not visible now that we don't support docker but they are still there. This disables the warnings about missing source maps